git_machete/tests/functional/test_machete.py

```diff
@@ -465,18 +472,18 @@ def test_branch_reappears_in_definition(self) -> None:
             \t\n
             develop
             """
-        expected_error_msg: str = fmt('.git/machete, line 5: branch `develop` re-appears in the tree definition. '
-                                      'Edit the definition file manually with `git machete edit`')
 
         self.repo_sandbox.new_branch("root")
         self.rewrite_definition_file(body)
 
-        machete_client = MacheteClient(git)  # Only to workaround sys.exit while calling launch(['status'])
-        try:
-            machete_client.read_definition_file()
```

Comment:
Removed the inconsistency here: some test methods (like this one) tested on the level of `MacheteClient` (which is CLI-agnostic), while most tested on the highest possible level - CLI itself (wrapped into `self.launch_command` call).

This PR unifies all tests to cover all the levels down from CLI in all test cases.